data/beps/beps.yml

```diff
@@ -279,16 +276,17 @@
     content:
     -   raw
     leads:
-    -   name: Sylvain Takerkart
-        email: sylvain.takerkart@univ-amu.fr
-    -   name: Julia Sprenger
-        email: julia.sprenger@univ-amu.fr
+    -   name: Yaroslav O. Halchenko
```

Comment:
@yarikoptic @bendichter 
I was told there has been a change of leadership here

Reply:
see multiline suggestion below